test/powershell/Modules/Microsoft.PowerShell.Utility/Set-Date.Tests.ps1

```diff
@@ -7,11 +7,17 @@ Describe "Set-Date" -Tag "CI" {
     }
 
     It "Set-Date should be able to set the date in an elevated context" -Skip:(! $IsElevated) {
-        { get-date | set-date } | Should not throw
+        { Get-Date | Set-Date } | Should Not Throw
+    }
+
+    It "Set-Date should be able to set the date with -Date parameter" -Skip:(! $IsElevated) {
+        $target = Get-Date
+        $expected = $target
+        Set-Date -Date $target | Should Be $expected
     }
 
     It "Set-Date should produce an error in a non-elevated context" -Skip:($IsElevated) {
-        { get-date | set-date} | should throw
-        $Error[0].FullyQualifiedErrorId | should be "System.ComponentModel.Win32Exception,Microsoft.PowerShell.Commands.SetDateCommand"
+        { Get-Date | Set-Date } | Should Throw
```

Comment:
Please change this to use ShouldBeErrorId

https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md

Reply:
OK😄